tests/test_functional.py

```diff
@@ -120,3 +124,48 @@ def test_invalid_input(self, user, testapp):
             assert "Import proběhl úspěšně" not in res
         assert Record.query.count() == existing_records
         assert Batch.query.count() == existing_batches
+
+
+class TestDonorsOverview:
+
+    TEST_IMPORTS_DF = pandas.read_csv(
```

Comment:
Je OK, že se toto děje v definici třídy a tedy i pokud Pytest tyto testy vůbec nespouští?

Reply:
Předěláme na fixturu.

Reply:
Jestli máš čas, mrkni na poslední fixup commit, kde jsem vytvořil funkci s pamětí, která ten dataframe vrací a pamatuje si výsledek, takže se to i při použití mimo fixture zavolá jen jednou. Teď je i ta fixtura trošku navíc, ale třeba se to bude ještě hodit.